apps/taxes/src/modules/avatax/ui/create-avatax-configuration.tsx

```diff
@@ -34,8 +53,18 @@ export const CreateAvataxConfiguration = () => {
 
   return (
     <AvataxConfigurationForm
-      isLoading={isCreateLoading}
-      onSubmit={submitHandler}
+      submit={{
+        isLoading: isCreateLoading,
+        handleFn: submitHandler,
+      }}
+      validateAddress={{
+        isLoading: validateAddressMutation.isLoading,
+        handleFn: validateAddressHandler,
+      }}
+      validateCredentials={{
+        isLoading: validateCredentialsMutation.isLoading,
+        handleFn: validateCredentialsHandler,
+      }}
```

Comment:
I think it should be memoized, your `useCallback` is useless now because you create object

Reply:
Memoized where? I am not sure I understand.

Reply:
You used `useCallback` on `validateCredentialsHandler`

but it doesnt matter because you create a component with prop


```
 validateCredentials={{
        isLoading: validateCredentialsMutation.isLoading,
        handleFn: validateCredentialsHandler,
      }}
```

so `validateCredentials={ {} }` is always re-rendering because new object literal is created

You need to put entire 
```
{
        isLoading: validateCredentialsMutation.isLoading,
        handleFn: validateCredentialsHandler,
      }
```

into useMemo or make it flat